src/else.js

```diff
@@ -14,7 +14,10 @@ export class Else extends IfCore {
   bind(bindingContext, overrideContext) {
     super.bind(bindingContext, overrideContext);
     // Render on initial
-    if (!this.ifVm.condition) {
+    if (this.ifVm.condition) {
+      this._hide();
+    }
+    else {
```

Comment:
According to ESLint, the `else` should be on the same line as the closing brace from the `if`.